[2024-06-06T15:15:26.010+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-06T15:15:26.023+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pg_loader.load_logs manual__2024-06-06T15:15:24.878362+00:00 [queued]>
[2024-06-06T15:15:26.028+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pg_loader.load_logs manual__2024-06-06T15:15:24.878362+00:00 [queued]>
[2024-06-06T15:15:26.028+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-06T15:15:26.036+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_logs> on 2024-06-06 15:15:24.878362+00:00
[2024-06-06T15:15:26.039+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=733) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-06T15:15:26.040+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pg_loader', 'load_logs', 'manual__2024-06-06T15:15:24.878362+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/load_pg_dag.py', '--cfg-path', '/tmp/tmpn727vz7j']
[2024-06-06T15:15:26.041+0000] {standard_task_runner.py:63} INFO - Started process 734 to run task
[2024-06-06T15:15:26.041+0000] {standard_task_runner.py:91} INFO - Job 16: Subtask load_logs
[2024-06-06T15:15:26.067+0000] {task_command.py:426} INFO - Running <TaskInstance: pg_loader.load_logs manual__2024-06-06T15:15:24.878362+00:00 [running]> on host 945bac6cf142
[2024-06-06T15:15:26.112+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pg_loader' AIRFLOW_CTX_TASK_ID='load_logs' AIRFLOW_CTX_EXECUTION_DATE='2024-06-06T15:15:24.878362+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-06T15:15:24.878362+00:00'
[2024-06-06T15:15:26.113+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-06T15:15:26.125+0000] {base.py:84} INFO - Using connection ID '***-etl' for task execution.
[2024-06-06T15:15:26.129+0000] {python.py:237} INFO - Done. Returned value was: [(1, 'default', 0.0, 0.0), (2, 'bronze', 0.05, 20000.0), (3, 'silver', 0.1, 50000.0), (4, 'gold', 0.15, 100000.0), (5, 'platinum', 0.2, 200000.0)]
[2024-06-06T15:15:26.130+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-06T15:15:26.141+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pg_loader, task_id=load_logs, run_id=manual__2024-06-06T15:15:24.878362+00:00, execution_date=20240606T151524, start_date=20240606T151526, end_date=20240606T151526
[2024-06-06T15:15:26.180+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-06T15:15:26.189+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-06T15:15:26.191+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
